open_spiel/python/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -184,8 +184,15 @@ def _watcher(*, config, num=None, **kwargs):
 
 def _init_bot(config, game, evaluator_, evaluation):
   """Initializes a bot."""
-  noise = None if evaluation else (config.policy_epsilon, config.policy_alpha)
-  return mcts.MCTSBot(
+  if config.search_type.lower() == "minimax":
```

Comment:
probably good to make this into an enum so the list of search types is known.

Reply:
Can you clarify what you mean by having them as enum ?